src/helpers/turbo.ts

```diff
@@ -0,0 +1,100 @@
+import { capture } from '@snapshot-labs/snapshot-sentry';
+import snapshot from '@snapshot-labs/snapshot.js';
+import db from './mysql';
+import { fetchWithKeepAlive } from './utils';
+
+type Space = {
+  id: string;
+  turbo_expiration: number;
+};
+
+const SCHNAPS_API_URL = process.env.SCHNAPS_API_URL;
+const NETWORK_PREFIX = process.env.NETWORK === '1' ? 's:' : 's-tn:';
+const RUN_INTERVAL = 10 * 1e3; // 10 seconds
+
+// Periodically sync the turbo status of spaces with the schnaps-api
+export async function trackTurboStatuses() {
+  if (!SCHNAPS_API_URL) return;
+
+  while (true) {
+    // Step 1: Query all the spaces from the schnaps-api
+    let spaces = await getSpacesExpirationDates();
+
+    spaces = spaces
+      .filter(space => space.id.startsWith(NETWORK_PREFIX))
```

Comment:
Schnaps api always return `[ { id: 'test.wa0x6e.eth', turbo_expiration: 0 } ]` without prefix `s:` 😅

Reply:
> Schnaps api always return `[ { id: 'test.wa0x6e.eth', turbo_expiration: 0 } ]` without prefix `s:` 😅

it does, but only for recent spaces. can not edit old data where spaces were passed without network prefix

Reply:
I've added a `TODO`, we can address this in a few months, when we will reach this limit